inference/app/inference.py

```diff
@@ -34,6 +41,32 @@ def allowed_filesize(filesize):
         return False
 
 
+class Models(object):
```

Comment:
this would also move to `model_registry`.
Create an abstract base class `Models`
Implement `DockerModels(Models):` in a separate PR.